modules/administration/pages/monitoring.adoc

```diff
@@ -145,136 +76,150 @@ zypper in golang-github-prometheus-prometheus
 systemctl enable --now prometheus
 ----
 . Check that the Prometheus interface is loading correctly.
-In your browser, navigate to the URL of the server where Prometheus is installed, and listen on port 9090 (for example, [literal]``http://example.com:9090``).
-
-
-
-=== Configuring Prometheus
-
-Prometheus requires some configuration to collect metrics and set up alarms, or to display metrics graphically in Grafana.
-You can configure Prometheus in the static configuration file at [path]``/etc/prometheus/prometheus.yml``.
-It is important to understand how this file is structured.
-For example:
+In your browser, navigate to the URL of the server where Prometheus is installed, on port 9090 (for example, [literal]``http://example.com:9090``).
 
+. Open the configuration file at [path]``/etc/prometheus/prometheus.yml`` and add the configuration block below. Make sure to replace `server.url` with your {productname} server URL and adjust `username` and `password` fields to match valid {productname} credentials.
++
 ----
+# {productname} self-health metrics
 scrape_configs:
-- job_name: 'suse-manager-server'
+- job_name: 'mgr-server'
   static_configs:
     - targets:
-      - 'suse-manager.local:9100'  # Node exporter
-      - 'suse-manager.local:9187'  # PostgreSQL exporter
-      - 'suse-manager.local:5556'  # JMX exporter (Tomcat)
-      - 'suse-manager.local:5557'  # JMX exporter (Taskomatic)
-      - 'suse-manager.local:9800'  # Taskomatic
+      - 'server.url:9100'  # Node exporter
+      - 'server.url:9187'  # PostgreSQL exporter
+      - 'server.url:5556'  # JMX exporter (Tomcat)
+      - 'server.url:5557'  # JMX exporter (Taskomatic)
+      - 'server.url:9800'  # Taskomatic
     - targets:
-      - 'suse-manager.local:80'    # Message queue
+      - 'server.local :80'    # Message queue
       labels:
         __metrics_path__: /rhn/metrics
+
+# Managed systems metrics:
+- job_name: 'mgr-clients'
+  uyuni_sd_configs:
+   - host: "http://server.url"
+     username: "admin"
+     password: "admin"
 ----
 
-For more information about configuring Prometheus, see the official Prometheus documentation at https://prometheus.io/docs/prometheus/latest/configuration/configuration/
+. Save the configuration file and restart the Prometheus service:
++
+----
+systemctl restart prometheus
+----
 
+[NOTE]
+For more information about the Prometheus configuration options, see the official Prometheus documentation at https://prometheus.io/docs/prometheus/latest/configuration/configuration/
```

Comment:
```suggestion
====
For more information about the Prometheus configuration options, see the official Prometheus documentation at https://prometheus.io/docs/prometheus/latest/configuration/configuration/.
====
```

Reply:
Not sure this really needs to be in an admonition, could it just be a paragraph on its own?

Reply:
Can be a paragraph on its own, yes.